src/commands/ImportCommand.js

```diff
@@ -0,0 +1,83 @@
+import fs from "fs";
+import path from "path";
+import child from "child_process";
+import Command from "../Command";
+import progressBar from "../progressBar";
+import PromptUtilities from "../PromptUtilities";
+
+export default class ImportCommand extends Command {
+  initialize(callback) {
+    const inputPath = this.input[0];
+
+    if (!inputPath) {
+      callback(new Error("Missing argument: Path to external repository"));
+    }
+
+    const externalRepoPath = path.resolve(inputPath);
+
+    try {
+      const stats = fs.statSync(externalRepoPath);
+      if (!stats.isDirectory()) {
+        throw new Error(`Input path "${inputPath}" is not a directory`);
+      }
+      const packageJson = path.join(externalRepoPath, "package.json");
+      const packageName = require(packageJson).name;
+      if (!packageName) {
+        throw new Error(`No package name specified in "${packageJson}"`);
+      }
+      this.targetDir = "packages/"+packageName;
```

Comment:
Yeah the style has been to usually have whitespace between operators (we should add `space-infix-ops`)